.vscode/settings.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
I feel like a lot of this should be handled by the linter so that its consistent across coding environments. Not that it couldn't be handled in the settings of VS code, but then we possibly have to track this in 2 places and the editor and linter could be at odds.